components/Op/OpDetailForm.js

```diff
@@ -397,8 +410,18 @@ class OpDetailForm extends Component {
                     />
                   )}
                 </Form.Item>
-                <Form.Item label={opEndDate}>
-                  {getFieldDecorator('endDate', {})(
+                <Form.Item
+                  label={opEndDate}
+                  name='End date'
+                >
+                  {getFieldDecorator('endDate', {
+                    rules: [
+                      {
+                        required: this.state.requiredForPublish,
+                        message: 'End date is required'
```

Comment:
end date is not required - even if a start date is present the end date can be left blank if the activity is a single day event.